TicTacToe2/TicTacToeGame.cs

```diff
@@ -8,7 +8,31 @@ class TicTacToeGame
     {
         public void Play()
         {
+            bool winner = false;
+            while (!winner)
+            {
+                DrawBoard();
+                UserInput();
+                winner = CheckWinDraw();
```

Comment:
CheckWinDraw seems like it would return true if the game has a winner and also if the game is a draw. Assigning this to a variable called winner seems somewhat confusing. I would recommend splitting this into two separate functions individually checking for win and draw.